docs/standard-library/built-in-data-types.md

```diff
@@ -4,4 +4,342 @@ toc_min_heading_level: 2
 toc_max_heading_level: 3
 ---
 
-See [Language Basics: Built-in types](../language-basics/built-in-types)
+# Data types
```

Comment:
This feels liek this has a bit of duplication with the `built-in-types` wondering if we should not omit all scalars and maybe `Array`

Reply:
It's supposed to. This is the autogenerated reference docs, rather than the manually written ones.

Reply:
I guess was removing them from the auto generation but maybe it’s fine to have the duplicate olwith maybe this one having more detailed doc